### DISCUSSION TITLE
Warning: A props object containing a "key" prop is being spread into JSX

### DISCUSSION THREAD

**Discussion by DrMalchev** [UI5/webcomponents-react#6062] [General]
It seems that AnalyticalTable is throwing this warning:
chunk-3L3Z2TQX.js?v=3fe2137a:64 Warning: A props object containing a "key" prop is being spread into JSX:
  let props = {key: someKey, role: ..., className: ..., style: ..., ref: ..., data-component-name: ..., children: ...};
  <div {...props} />
React keys must be passed directly to JSX without using spread:
  let props = {role: ..., className: ..., style: ..., ref: ..., data-component-name: ..., children: ...};
  <div key={someKey} {...props} />
    at http://localhost:5173/node_modules/.vite/deps/@ui5_webcomponents-react.js?v=3fe2137a:67146:18
    at div
    
I really cant figure out what can be the issue.
Any help is appreciated.
Thanks!

**Comment by Lukas742** [comment]
Hi @DrMalchev 

with React19 spreading React keys is not supported anymore, that's why React introduced warnings with their 18.3 version to let developers know that this must be adjusted for React 19. We will fix these warnings before React releases v19, but currently this has a low priority as it will not affect our components. 
If you don't want to see these warnings, you can use React 18.2, as React 18.3 only added deprecation warnings like this and no new features or bug fixes.

**Reply by DrMalchev** [reply]
Thanks!
